### ISSUE TITLE
Zigbee remove command not presente in list

### ISSUE THREAD

**Issue by maury77** [AlexxIT/XiaomiGateway3#841]
I have problem with one bulb , the bulb is not visible 

[MQTT] log/z3 b'6 5CB6:  842E14FFFE38624D 0  UNRESPONSIVE 1298\r'

I want to try to reset device e remove fromm gateway , but command remove is not present ...

![image](https://user-images.githubusercontent.com/35746732/195200525-747be6cd-e171-49de-a1c8-707d4323e4fd.png)

**Comment by WarC0zes** [comment]
Hi,
I would like to remove a Sonoff snzb-04 door sensor. But as on Mihome we can't see it, I can't delete it from the application and in HA the Zigbee remove option is no longer there.
Is there a solution?

**Comment by DJTerentjev** [comment]
Maybe it will work for you?
![image](https://user-images.githubusercontent.com/1267473/210131384-68931522-85c1-4e14-87c4-64d0400f3860.png)

**Comment by WarC0zes** [comment]
> Maybe it will work for you? ![image](https://user-images.githubusercontent.com/1267473/210131384-68931522-85c1-4e14-87c4-64d0400f3860.png)

Yes, It's work. I had not thought to go to the equipment page, well seen. 
Thanks
